datalad/support/tests/test_external_versions.py

```diff
@@ -25,12 +25,17 @@
     swallow_logs,
 )
 
-from unittest.mock import patch
 from nose.tools import (
-    assert_true, assert_false,
-    assert_equal, assert_greater_equal, assert_greater,
-    assert_raises, assert_in
+    assert_equal,
+    assert_false,
+    assert_greater,
+    assert_greater_equal,
+    assert_in,
+    assert_not_in,
+    assert_raises,
+    assert_true,
 )
+from unittest.mock import patch
```

Comment:
Aside from trying to introduce conflicts :), why move this `patch` import?  My first guess was that you were sorting somehow, but that doesn't seem to be the case because `nose.SkipTest` import is below.

Reply:
;-) yeah, who knows.  I will RF as where it belongs in master version ;)